pkg/operation/botanist/component/istio/charts/istio/istio-istiod/templates/service.yaml

```
@@ -3,8 +3,10 @@ kind: Service
 metadata:
   name: {{ .Values.serviceName }}
   namespace: {{ .Release.Namespace }}
-{{- if .Values.ignoreMode }}
   annotations:
+    networking.resources.gardener.cloud/from-world-to-ports: '[{"port":{{ toString .Values.ports.https }},"protocol":"TCP"}]'
```

```suggestion
    networking.resources.gardener.cloud/from-world-to-ports: '[{"port":{{ .Values.ports.https }},"protocol":"TCP"}]'
```

This should not be needed, or?